kerlescan/audit_logging.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import json
+import base64
+
+from kerlescan.service_interface import get_key_from_headers
+
+AUDIT_LEVEL_NAME = "AUDIT"
+AUDIT_LEVEL_NUM = int((logging.INFO + logging.WARNING) / 2)
+
+
+def audit(self, message, *args, **kws):
+    if self.isEnabledFor(AUDIT_LEVEL_NUM):
+        request = kws.pop("request", None)
+
+        status = None
+        status = "success" if kws.pop("success", False) else None
+        status = "failure" if kws.pop("failure", False) else None
```

Comment:
This requires the caller to provide success=True or failure=True.  Would it make more sense for the caller to provide just success=[True|False] and then set status = "success" or "failure" based on the value True or False?